content/sigs/docs/gsod/ideas.adoc

```diff
@@ -34,8 +34,8 @@ We encourage proposals based on a technical writer's expertise and interests.
 | Document Jenkins on Kubernetes
 | AsciiDoc, Guide, Compilation, Kubernetes
 | Jenkins on Kubernetes is a popular theme for Jenkins users.
-  There were a lot of presentations and articles about running Jenkins on Kubernetes, but we need a central location for documentation describing Jenkins on Kubernetes.
-  We would like to create new documentation which would describe the concepts, techniques, and choices for Kubernetes users running Jenkins.
+  There were a lot of presentations and articles about running Jenkins on Kubernetes, but we need to expand the documentation describing Jenkins on Kubernetes.
+  We would like to create documentation that describes the concepts, techniques, and choices for Kubernetes users running Jenkins.
```

Comment:
Reference the 2020 project for the background context?
+1 for keeping the idea, there is a lot more docs that could be added

Reply:
Great idea.  Added in dff2225532871d645323f1c0cc0812ab9bd31018